x/upgrade/keeper/keeper.go

```diff
@@ -44,14 +46,16 @@ type Keeper struct {
 // cdc - the app-wide binary codec
 // homePath - root directory of the application's config
 // vs - the interface implemented by baseapp which allows setting baseapp's protocol version field
-func NewKeeper(skipUpgradeHeights map[int64]bool, storeKey storetypes.StoreKey, cdc codec.BinaryCodec, homePath string, vs xp.ProtocolVersionSetter) Keeper {
+func NewKeeper(skipUpgradeHeights map[int64]bool, storeKey storetypes.StoreKey, cdc codec.BinaryCodec, homePath string, vs xp.ProtocolVersionSetter, authKeeper types.AccountKeeper, authority sdk.AccAddress) Keeper {
```

Comment:
Gov and group keepers (and soon staking keeper #10988) have as last argument a `Config` object. Should we do the same here? We can set the default to be the gov address.